critical_demand.py

```diff
@@ -541,6 +539,7 @@ def run_simulation(df_costs, data, settings):
     ) + non_critical_demand[sequences_demand.index].sum(axis=0)
 
     total_opex_costs = asset_results.total_opex_costs.sum() * project_lifetime
+    overall_peak_demand = sequences_demand.max() + sequences_critical_demand.max()
```

Comment:
if you sum the max of the 2 sequences it is likely to be larger than the max of the sum

Reply:
That is true.